pkg/response/README.md

```diff
@@ -0,0 +1,34 @@
+# Event Logger
+
+This package enables response with event logging in go-restful apps.
+For the event logging please read logger/event package
```

Comment:
This part of the docs is not right, copy and pasted?

Reply:
Sorry i missed it. i will change it